### ISSUE TITLE
docker_container is not splitting labels that have ':'(s) in the value.

### ISSUE THREAD

**Issue by david-caldwell** [chef-cookbooks/docker#965]
### Cookbook version
2.17.0

### Chef-client version
13.6.4

### Platform Details
Ubuntu 16.04.3 x64 running on DigitalOcean Cloud Provider

### Scenario:
I'm attempting to pass in a `Label` with a value that has colons ` : ` in it. The entire string is split by ` : ` and you end up with the second value of the array as your value. Everything after the second colon is discarded.

### Steps to Reproduce:
Pass in a label like `labels ['HealthCheckArgs:http://{{ host }}:{{ tcp 8080 }}/health_check']`

### Expected Result:
HealthCheckArgs=http://{{ host }}:{{ tcp 8080 }}/health_check

### Actual Result:
HealthCheckArgs=http

Changing line 26 in libraries/helpers_base.rb in the following way resolves the issue.
```
-    h[parts[0]] = parts[1]
+    h[parts[0]] = parts[1..-1].join(':')
```

**Comment by spuder** [comment]
I work with David and can confirm that Labels does not work when a string contains multiple colons`:`

     'HealthCheckArgs:http://{{ host }}:{{ tcp 8080 }}/health_check'

becomes

    ['HealthCheckArgs', 'http', '//{{ host }}',{{ tcp 8080 }}/health_check'],

Which gets truncated to 

      'HealthCheckArgs:'http'

The work around is to provide labels as a hash, or fix the `coerce_labels` method.

**Comment by chasebolt** [comment]
fixed in `v4.0.1`
